File: Irrgarten/src/irrgarten/main/GamePlay.java
package irrgarten.main;
import irrgarten.Game;
import irrgarten.controller.Controller;
import irrgarten.UI.TextUI;

/**
 *
 * @author Daniel Hidalgo Chica and Luis Esteban Valdivieso
 */
public class GamePlay {
    static final int N_PLAYERS=4;
    public static void main (String [] args){
        
        Game game = new Game(N_PLAYERS);
        TextUI textUI = new TextUI();
        Controller controller = new Controller(game, textUI);
        controller.play();
    }
}

File: Irrgarten_ruby/GamePlay/gameplay.rb
#encoding=utf-8
require_relative "../controller/controller.rb"
require_relative "../textUI/textUI.rb"
require_relative "../game.rb"
class GamePlay
    @N_PLAYERS = 1
    # Main method to start the game
    def self.main
        game = Irrgarten::Game.new(@N_PLAYERS)
        text_ui = UI::TextUI.new
        controller = Control::Controller.new(game, text_ui)
        controller.play
    end
end

GamePlay.main